mailers/newMessageMailer.ts

```diff
@@ -39,14 +40,17 @@ export function newMessageMailer({ to, chatid, from, messageContent }: NewMessag
       <p>${chatUrl}</p>
     `,
   }
+  const log: Logger = new Logger({ name: "mailer" })
 
   return {
     async send() {
       if (process.env.NODE_ENV === "production") {
         await smtp.sendMail(msg)
+        log.info("An email was sent.")
       } else {
         // Preview email in the browser
         await previewEmail(msg)
+        log.info("An preview mail was created.")
```

Comment:
Maybe we should print different log statements after a (preview)Mail was created/sent because of a new message or because of resetPassword feature. This way we could debug easier